xds/src/main/java/io/grpc/xds/AbstractXdsClient.java

```diff
@@ -71,6 +73,7 @@ final class AbstractXdsClient {
   private final BackoffPolicy.Provider backoffPolicyProvider;
   private final Stopwatch stopwatch;
   private final Node bootstrapNode;
+  private final XdsClient.TimerLaunch timerLaunch;
```

Comment:
Not related to this PR but I looked at this class `AbstractXdsClient` and:

- the comment on lines 60-61 is (now) wrong: this is not a base type for any XdsClient implementations
- any var of this type is called `xdsChannel` so the type should also be called XdsChannel or such since it really encapsulates Xds channel functionality

Not asking these changes to be made in this PR but reviewing code becomes difficult because of misleading comments/names etc.

Reply:
There is a TODO on line 92 about renaming this class.  That should be its own PR.

Reply:
Okay